pkg/queue/error.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+//
+// Description: error
+package queue
+
+import (
+	"fmt"
+)
+
+// NewMaxCapacityError
+func NewMaxCapacityError(capacity int) *MaxCapacityError {
+	return &MaxCapacityError{
+		capacity: capacity,
+	}
+}
+
+var _ error = new(MaxCapacityError)
+
+// MaxCapacityError
```

Comment:
Please provide more descriptive comments than just the including the function name to 'game' the linter, these should be indicative of why/what this is. Consider it to be justifying its existence. 

This applies to all comments in this PR.